locale/README.md

```diff
@@ -28,3 +28,17 @@ verify locale
 ```sh
 yarn locale:verify
 ```
+
+
+## Locale specific strings
+
+The English resource (string) container [en.json](./locales/en.json) on `main` branch is monitored for changes. All modifications will automatically be picked up and propagated to the other locale files by the Globalization Services team at Qlik at regular intervals (normally at least twice a week).
+
+## Pull requests touching English string resources
+
+Please request a review from a technical writer on all PRs touching English string resources, to ensure changes are consistent with style guides and translatability requirements.
+The primary contact is the technical writer assigned to a HLP (documentation) task on the related PS (product scenario). If unsure, please ask in [slack: #content-and-media](https://qlikdev.slack.com/archives/C05642TAY)
```

Comment:
Do we want to mention this slack channel in an open source repository? Not everyone can reach it.